wp-content/themes/reconasia/footer.php

```diff
@@ -15,18 +15,25 @@
 			<footer id="site-footer" class="footer" role="contentinfo">
 				<a href="https://www.csis.org" class="footer__logo"><?php include( get_template_directory() . '/assets/static/csis-logo.svg'); ?></a>
 
-				<?php dynamic_sidebar( 'sidebar-1' ); ?>
-				<?php dynamic_sidebar( 'sidebar-2' ); ?>
-
-				<div class="footer__contact">
-					<address class="footer__address">
-						1616 Rhode Island Ave NW<br />
-						Washington, DC 20036
-					</address>
-
-					<p class="footer__phone">(202) 887-0200</p>
-
-					<?php dynamic_sidebar( 'social-share' ); ?>
+				<div class=footer__desc>
+					<?php dynamic_sidebar( 'sidebar-1' ); ?>
+					<div class="footer__desc-line"></div>
+					<?php dynamic_sidebar( 'sidebar-2' ); ?>
+				</div>
+				<?php dynamic_sidebar( 'sidebar-3' ); ?>
+
+				<!-- Social Share -->
+				<div class="footer__social">
+					<p class="footer__social-header">
+						Follow Us
+					</p>
+					<div class="footer__social-icons">
+						<a href="https://www.facebook.com/reconnasia/" class="footer__facebook-icon"><?php echo reconasia_get_svg( 'facebook' ); ?></a>
```

Comment:
Can you add `aria-labels` to each of these that describes what the link is doing? Eg. `aria-label="Visit the Reconnecting Asia Facebook page"`